website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -616,11 +616,13 @@ This endpoint lists workspaces in the organization.
 
 This endpoint supports pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
 
-| Parameter      | Description                                                                |
-| -------------- | -------------------------------------------------------------------------- |
-| `page[number]` | **Optional.** If omitted, the endpoint will return the first page.         |
-| `page[size]`   | **Optional.** If omitted, the endpoint will return 20 workspaces per page. |
-| `search[name]` | **Optional.** Allows searching the organization's workspaces by name.      |
+| Parameter              | Description                                                                            |
+| ---------------------- | -------------------------------------------------------------------------------------- |
+| `page[number]`         | **Optional.** If omitted, the endpoint will return the first page.                     |
+| `page[size]`           | **Optional.** If omitted, the endpoint will return 20 workspaces per page.             |
+| `search[name]`         | **Optional.** Allows searching the organization's workspaces by name.                  |
+| `search[tags]`         | **Optional.** The list of tags to match when searching an organization's workspaces.   |
+| `search[exclude-tags]` | **Optional.** The list of tags to exclude when searching an organization's workspaces. |
```

Comment:
We should mention allow multiple tags and exclude tags using comma-separated values, and how multiple values behaves.

Reply:
done in the new PR link: https://github.com/hashicorp/terraform-docs-common/pull/54